spec/cluster_spec.rb

```diff
@@ -0,0 +1,35 @@
+#
+# Cookbook Name:: rabbitmq
+# Spec:: cluster
+#
+# Author: Sunggun Yu <sunggun.dev@gmail.com>
+# Copyright (C) 2015 Sunggun Yu
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'spec_helper'
+
+describe 'rabbitmq::cluster' do
+  let(:runner) { ChefSpec::ServerRunner.new(REDHAT_OPTS) }
+  let(:node) { runner.node }
+  let(:chef_run) do
+    runner.converge(described_recipe)
+  end
+
```

Comment:
pretty think specs for all that lwrp code, hopefully more will be comming for this new support.